identity/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libp2p-identity"
-version = "0.2.4"
+version = "0.2.5"
```

Comment:
We use workspace inheritance in the root `Cargo.toml`, can you also bump the referenced version there please?